### PR TITLE
CC-61 Make PreviewBlock more suitable for all image sizes

### PR DESCRIPTION
**Pull request by ArtsemiLasyi**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
-    <ThemeProvider theme={theme}>
-      <GlobalStyles />
-      <div className="App">
-        <Sidebar />
-        <Content />
-      </div>
-    </ThemeProvider>
-  )
-
+    return (
```
Comment (PavelPyasecky) [inline src/App.tsx:11]: @ArtsemiLasyi Did you use some Linter or this is your own code style?

File: src/constants/colors.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const PRIMARY_COLOR = "#F0A202";
```
Comment (PavelPyasecky) [inline src/constants/colors.ts]: Please add empty end line to the file

**Review by PavelPyasecky — APPROVED** [review]
That looks really good! And I find the PreviewBlock component pretty nice. In my view, there is no need to pay a lot of attention to design details like button `Расписание Богослужений` until we've got a design requirements. Thanks @ArtsemiLasyi !

File: src/App.tsx (hunk 1 above)
Comment (ArtsemiLasyi) [reply src/App.tsx:11]: Yep, my own style code

**Review by 17Sergey — APPROVED** [review]

**Pull request merged**